### PR TITLE
fix(sdk): replace rmSync for better compatibility

### PR DESCRIPTION
**Pull request by amateima**
Signed-off-by: amateima <amatei@umaproject.org>

**Motivation**

https://app.shortcut.com/uma-project/story/5250/adjust-the-sdk-build-script-to-not-use-fs-rmsync

**Summary**

I replaced the `rmSync` function call with functions that were added in older versions of NodeJS. The build script works now with NodeJS versions < v14.14

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [x]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [x]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: packages/sdk/build.js [hunk 1]
```diff
@@ -16,6 +16,23 @@ const moveFile = (from, to, filename) => {
   fs.renameSync(path.resolve(...from), path.resolve(...to));
 };
 
+// A deep deletion function for node like `rm -rf` which works for versions older than v14.14
+const rimraf = function (directoryPath) {
```
Comment (daywiss) [inline packages/sdk/build.js:20]: very interesting, didnt realize it was this complicated to remove directory.

**Review by daywiss — APPROVED** [review]
looks good, tested this by building sdk

**Review by mrice32 — APPROVED** [review]
Looks good! One minor question

File: packages/sdk/build.js [hunk 2]
```diff
@@ -16,6 +16,23 @@ const moveFile = (from, to, filename) => {
   fs.renameSync(path.resolve(...from), path.resolve(...to));
 };
 
+// A deep deletion function for node like `rm -rf` which works for versions older than v14.14
```
[the remark this answers is not in the document]
Comment (amateima) [reply packages/sdk/build.js:19]: Chandler had this problem while building the SDK locally

**Pull request merged**